integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -122,6 +122,12 @@ object BasicTests extends TestSuite{
       val evaled = exec('basic / "Resources.sc")
       assert(evaled.out.string.contains("1745"))
     }
+    'testSilentIvy{
+      val evaled1 = exec('basic/"scalaTags.sc")
+      assert(evaled1.err.string.contains("scalatags"))
+      val evaled2 = exec('basic/"scalaTags.sc", "-s")
+      assert(!evaled2.err.string.contains("scalatags"))
+    }
```

Comment:
Can you check for a longer string in your `contains` to make it more obvious what it's checking? Something like the string `resolving dependencies` would make it obvious it's checking for the text that talks about resolving dependencies